lib/cartopy/crs.py

```diff
@@ -1329,13 +1310,10 @@ def __init__(self, central_longitude=0.0, globe=None):
         proj4_params = [('proj', 'mill'), ('lon_0', central_longitude)]
         # See Snyder, 1987. Eqs (11-1) and (11-2) substituting maximums of
         # (lambda-lambda0)=180 and phi=90 to get limits.
+        self.threshold = 0.5
```

Comment:
That's the default now, so can be removed?

Reply:
Removed